dist/src/main/java/io/camunda/zeebe/shared/management/BackupEndpoint.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+package io.camunda.zeebe.shared.management;
+
+import io.camunda.zeebe.gateway.admin.backup.BackupApi;
+import io.camunda.zeebe.gateway.admin.backup.BackupRequestHandler;
+import io.camunda.zeebe.gateway.impl.broker.BrokerClient;
+import io.camunda.zeebe.util.VisibleForTesting;
+import java.util.concurrent.CompletionException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.actuate.endpoint.annotation.Selector;
+import org.springframework.boot.actuate.endpoint.annotation.WriteOperation;
+import org.springframework.boot.actuate.endpoint.web.WebEndpointResponse;
+import org.springframework.boot.actuate.endpoint.web.annotation.WebEndpoint;
+import org.springframework.lang.NonNull;
+import org.springframework.stereotype.Component;
+
+@Component
+@WebEndpoint(id = "backups", enableByDefault = false)
+final class BackupEndpoint {
+  private final BackupApi api;
+
+  @SuppressWarnings("unused") // used by Spring
+  @Autowired
+  public BackupEndpoint(final BrokerClient client) {
+    this(new BackupRequestHandler(client));
+  }
+
+  BackupEndpoint(final BackupApi api) {
+    this.api = api;
+  }
+
+  @WriteOperation
+  public WebEndpointResponse<?> take(@Selector @NonNull final long id) {
+    try {
+      final long backupId = api.takeBackup(id).toCompletableFuture().join();
+      return new WebEndpointResponse<>(new TakeBackupResponse(backupId));
+    } catch (final CompletionException e) {
+      return new WebEndpointResponse<>(
+          new TakeBackupError(id, e.getCause().getMessage()),
+          WebEndpointResponse.STATUS_INTERNAL_SERVER_ERROR);
+    } catch (final Exception e) {
+      return new WebEndpointResponse<>(
+          new TakeBackupError(id, e.getMessage()),
+          WebEndpointResponse.STATUS_INTERNAL_SERVER_ERROR);
+    }
+  }
+
+  @VisibleForTesting
+  record TakeBackupResponse(long id) {}
+
+  @VisibleForTesting
+  record TakeBackupError(long id, String failure) {}
```

Comment:
I'm not so sure about that. Serializing stack traces is useful for us, but not so much for users, who will be using this API for self managed. We've documented the error format we would like in the future in the original issue, which is also not an exception. I would rather go for that, or explore `problem+json` as a format, instead of serializing exceptions.